magit-gh-comments-diff.el

```diff
@@ -0,0 +1,25 @@
+(defconst magit-gh--hunk-header-re "@@ -\\([0-9]+\\),\\([0-9]+\\) \\+\\([0-9]+\\),\\([0-9]+\\) @@")
+
+(defun magit-gh--jump-to-file-in-diff (file)
+  (if-let ((result (or (re-search-forward (format "^\\(copy\\|rename\\) to %s$" file) nil t)
+                       (re-search-forward (format "+++ b/%s" file) nil t))))
+      result
+    (error "Could not find file '%s' in diff!" file)))
+
+(defun magit-gh--paint-diff ()
```

Comment:
Note to self: this needs some work - it's only highlighting the text. It should highlight the margins, up to a fixed width.